inference-engine/tests/functional/plugin/gpu/shared_tests_instances/behavior/ov_infer_request/inference_chaining.cpp

```diff
@@ -0,0 +1,32 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "behavior/ov_infer_request/inference_chaining.hpp"
+#include "common_test_utils/test_constants.hpp"
+
+using namespace ov::test::behavior;
+
+namespace {
+
+const std::vector<std::map<std::string, std::string>> configs = {
+    {}
+};
+
+const std::vector<std::map<std::string, std::string>> HeteroConfigs = {
+    {{"TARGET_FALLBACK", CommonTestUtils::DEVICE_GPU}}
+};
+
+INSTANTIATE_TEST_SUITE_P(smoke_BehaviorTests, OVInferenceChaining,
+                        ::testing::Combine(
+                                ::testing::Values(CommonTestUtils::DEVICE_GPU),
+                                ::testing::ValuesIn(configs)),
```

Comment:
BTW, does it work on GPU? `OVInferenceChaining` suppose dynamic cases as well:
- `StaticOutputToDynamicInput`
- `DynamicInputToDynamicOutput`
- `DynamicOutputToDynamicInput`
While GPU can support:
- `StaticOutputToStaticInput`

Reply:
Dynamic tests don't work indeed. I removed them from this PR and kept only minimal changes in infer request impl to make remote blob tests work. Other thing are to be addressed in https://github.com/openvinotoolkit/openvino/pull/8358